src/index.html

```diff
@@ -10,7 +10,51 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="cover"
+    >
+      <label for="bigForm" class="form bigForm">
+        <img
+         src="images/Search.svg"
+         alt="Search icon"
+         class="icon iconBig"
+        >
+        <input
+          type="text"
+          name="bigForm"
+          id="bigForm"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="input inputBig"
```

Comment:
the same, [checklist p.2](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Reply:
thank you, made the changes, try to push